### ISSUE TITLE
Should be able to send a reason to user if their organization has been rejected

### ISSUE THREAD

**Issue by NatalieEO** [dockstore/dockstore#2457]
## Feature Request 
Right now there isn't a way for a curator to let a user know why their organization was rejected. If the reason is small/easy to change, we should really let the user know about it so they can try again. For example, if I reject an org because the name is too general, they're going to have no idea why.
I'd suggest either something through Dockstore or requiring the email field in the org form.

**Comment by agduncan94** [comment]
This should be pretty straightforward, not sure if we would want this in hotfix or wait till 1.7.0.

**Comment by denis-yuen** [comment]
Email seems simplest, let's get that required. 
Shoots two birds with one stone (i.e. we get a verified email so we have additional confidence they are who they say they are and we get a more guaranteed point of contact for the org)

**Comment by agduncan94** [comment]
So is the SOP just to email the provided email with the reasoning for rejection?

**Comment by denis-yuen** [comment]
Yeah, and reject the org if there's no valid email. Which does bring up the question of how to contact a person with no valid email...
